docs/security/trust/content_trust.md

```diff
@@ -267,13 +268,18 @@ Because the tag `docker/cliffs:latest` is not trusted, the `pull` fails.
 A user who wants to disable content trust for a particular operation can use the
```

Comment:
Are you saying, combining this section with https://github.com/mansinahar/docker/blob/df89878f5bda496b9e49cf008ac9483491a69c11/docs/security/trust/content_trust.md#enable-and-disable-content-trust-per-shell-or-per-invocation ?